package.json

```diff
@@ -1,95 +1,29 @@
 {
-  "name": "kyt",
-  "version": "0.3.0",
-  "description": "kyt is a toolkit that encapsulates and manages the configuration for web apps.",
-  "main": "",
-  "bin": {
-    "kyt": "cli/index.js"
-  },
-  "engines": {
-    "node": ">=6.1.0"
+  "private": true,
```

Comment:
means it won't let you publish to npm, it's just as a safeguard. I think its nice to have these at the root level because you can run all the tests and linting at once.